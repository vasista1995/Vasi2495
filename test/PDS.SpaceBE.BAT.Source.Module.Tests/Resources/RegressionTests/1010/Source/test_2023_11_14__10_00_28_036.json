{
  "Device": "S0946T",
  "ParameterClass": "NON_SPC",
  "CFComment": "FOL",
  "Segment": "WB",
  "FourDReport": "*",
  "SpaceRawValues": [
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 240186690,
      "Seqnr": 1,
      "Value": 131.78,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 240186690,
      "Seqnr": 2,
      "Value": 132.22,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 240186690,
      "Seqnr": 5,
      "Value": 132.16,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 240186690,
      "Seqnr": 4,
      "Value": 127.74,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 240186690,
      "Seqnr": 3,
      "Value": 130.8,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    }
  ],
  "UniqueSourceKey": "BAT:BE:RU342051B01:BE1I:Wire_Bond1:CopperBallDiameter:67614:240186690:0",
  "PKey": "BAT:BE:RU342051B01:BE1I:Wire_Bond1:CopperBallDiameter:67614:240186690:0",
  "ParameterName": "CopperBallDiameter",
  "ParameterUnit": "um",
  "SampleDate": "2023-11-14T13:37:20Z",
  "SampleDateUtc": "2023-11-14T06:37:20Z",
  "SampleSize": 5,
  "Sigma": "1,87696563633968",
  "Mean": "130,94",
  "Median": "131,78",
  "Min": "127,74",
  "Max": "132,22",
  "Q1": "130,8",
  "Q3": "132,16",
  "LdsID": "1010",
  "IsCalcParam": 0,
  "MeanCntrlLow": 125.0,
  "MeanCntrlTarget": 145.0,
  "MeanCntrlHigh": 165.0,
  "RangeCntrlTarget": 0.0,
  "RangeCntrlHigh": 50.0,
  "SpecLow": 120.0,
  "SpecTarget": 145.0,
  "SpecHigh": 170.0,
  "CtrlMeanHighEnabled": "Y",
  "CtrlMeanLowEnabled": "Y",
  "CtrlMeanTargetEnabled": "Y",
  "CtrlLowEnabled": "N",
  "CtrlHighEnabled": "N",
  "CtrlRangeLowEnabled": "N",
  "CtrlRangeHighEnabled": "Y",
  "CtrlSigmaLowEnabled": "N",
  "CtrlSigmaCenterEnabled": "N",
  "CtrlRangeCenterEnabled": "Y",
  "CtrlCenterEnabled": "N",
  "CtrlSigmaHighEnabled": "N",
  "SpecLowEnabled": "Y",
  "SpecHighEnabled": "Y",
  "CreatedDate": "2023-11-14T13:37:20Z",
  "CreatedDateUtc": "2023-11-14T06:37:20Z",
  "UpdatedDate": "2023-11-14T13:37:20Z",
  "UpdatedDateUtc": "2023-11-14T06:37:20Z",
  "RvStoreFlag": "Y",
  "Operation": "Wire_Bond1",
  "Facility": "BE1I",
  "ChannelId": "67614",
  "CkcId": "0",
  "ChannelName": "Ball_Diameter/CU/50um",
  "SpaceInstanceName": "BE",
  "Sitekey": "BAT",
  "CalcCreated": "11/13/2023 22:19:04",
  "ILdsID": 0,
  "AcceptFlag": "Y",
  "UpdatedBy": " ",
  "SpecTargetOrigin": "V",
  "ViolCount": 0,
  "ExtMaMV": 129.57675,
  "ExtEwmaMV": 129.924492209842,
  "ExtMsMV": 1.12288556852423,
  "OutOfOrder": "N",
  "EwmaS": 1.67293980964953,
  "EwmaR": 4.0098453921013,
  "Calc2ID": 2525350,
  "LimitEnable": "YYNYNNYNYYNNNNNNNNNNNNNNNNNNNNNNNNNNN",
  "ExtEwmaMVLCL": 12.8656531239446,
  "ExtEwmaMVCenter": 15.268504,
  "ExtEwmaMVUCL": 17.6713548760554,
  "ExtMaMVLCL": 13.5694318513612,
  "ExtMaMVCenter": 15.268504,
  "ExtMaMVUCL": 16.9675761486388,
  "EwmaSLCL": 0.572474202225081,
  "EwmaSCenter": 1.51113357572865,
  "EwmaSUCL": 2.44979294923222,
  "EwmaRLCL": 1.38119890295338,
  "EwmaRCenter": 3.62764,
  "EwmaRUCL": 5.87408109704662,
  "ExtMSMVLCL": 0.00259778821504498,
  "ExtMSMVCenter": 1.2250389281796,
  "ExtMSMVUCL": 4.92102881621818,
  "ClOrigin": 0,
  "MVLal": 0.0,
  "MVUal": 0.0,
  "RawLal": 5.64136422726342,
  "RawUal": 24.8956437727366,
  "SigmaLal": 0.0,
  "SigmaUal": 0.0,
  "RangeLal": 0.0,
  "RangeUal": 0.0,
  "ExtSpecLower": 0.0,
  "ExtSpecUpper": 0.0,
  "ExtSpecLimEnable": "NN",
  "CalculationStrategy": "ND",
  "ChannelType": "N",
  "ChState": "Online"
}