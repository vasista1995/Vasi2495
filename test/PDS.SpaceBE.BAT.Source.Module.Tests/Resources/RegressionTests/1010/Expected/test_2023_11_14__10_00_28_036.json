{
  "IdSource": "SPACEACT2:BAT:BE:RU342051B01:BE1I:Wire_Bond1:CopperBallDiameter:67614:240186690:0",
  "Data_flat_Metadata": {
    "Device": "S0946T",
    "ParameterClass": "NON_SPC",
    "CFComment": "FOL",
    "Segment": "WB",
    "FourDReport": "*",
    "ParameterName": "CopperBallDiameter",
    "ParameterUnit": "um",
    "ParameterOper": "Wire_Bond1",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "ChannelId": "67614",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-11-14T06:37:20Z",
    "CreatedTimestamp": "2023-11-14T13:37:20Z",
    "UpdatedTimestamp": "2023-11-14T13:37:20Z",
    "UpdatedTimestampUtc": "2023-11-14T06:37:20Z",
    "ExportedTimestampUtc": "2023-11-14T13:37:20Z",
    "ExportedTimestamp": "2023-11-14T13:37:20Z",
    "SampleTimestampUtc": "2023-11-14T06:37:20Z",
    "SampleTimestamp": "2023-11-14T13:37:20Z",
    "SourceDataLevel": "C",
    "ChannelName": "Ball_Diameter/CU/50um",
    "RvStoreFlag": "Y",
    "CalcCreated": "11/13/2023 22:19:04",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 2525350,
    "ViolCount": 0,
    "CalculationStrategy": "ND",
    "LdsID": "1010",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 170.0,
        "SpecLow": 120.0,
        "SpecTarget": 145.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYNNYNYYNNNNNNNNNNNNNNNNNNNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "130,94",
        "Median": "131,78",
        "Sigma": "1,87696563633968",
        "Min": "127,74",
        "Max": "132,22",
        "Q1": "130,8",
        "Q3": "132,16"
      },
      "ControlLimits": {
        "ctrl_mean_low": 125.0,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 145.0,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 165.0,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "CtrlSigmaHighEnabled": "N",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 0.0,
        "ctrl_range_high": 50.0,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 129.57675,
        "ExtEwmaMV": 129.924492209842,
        "ExtMsMV": 1.12288556852423,
        "EwmaS": 1.67293980964953,
        "EwmaR": 4.0098453921013,
        "ExtEwmaMVLCL": 12.8656531239446,
        "ExtEwmaMVCenter": 15.268504,
        "ExtEwmaMVUCL": 17.6713548760554,
        "ExtMaMVLCL": 13.5694318513612,
        "ExtMaMVCenter": 15.268504,
        "ExtMaMVUCL": 16.9675761486388,
        "EwmaSLCL": 0.572474202225081,
        "EwmaSCenter": 1.51113357572865,
        "EwmaSUCL": 2.44979294923222,
        "EwmaRLCL": 1.38119890295338,
        "EwmaRCenter": 3.62764,
        "EwmaRUCL": 5.87408109704662,
        "ExtMSMVLCL": 0.00259778821504498,
        "ExtMSMVCenter": 1.2250389281796,
        "ExtMSMVUCL": 4.92102881621818,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 5.64136422726342,
        "RawUal": 24.8956437727366,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 240186690,
        "Seqnr": 1,
        "Value": 131.78,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240186690,
        "Seqnr": 2,
        "Value": 132.22,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240186690,
        "Seqnr": 5,
        "Value": 132.16,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240186690,
        "Seqnr": 4,
        "Value": 127.74,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240186690,
        "Seqnr": 3,
        "Value": 130.8,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:BAT:BE:BE1I:Wire_Bond1:CopperBallDiameter:67614",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "BAT:BE:RU342051B01:BE1I:Wire_Bond1:CopperBallDiameter:67614:240186690:0"
  }
}